pkg/api/steve/expose/expose.go

```diff
@@ -1,4 +1,4 @@
-package disallow
+package expose
```

Comment:
This name is a little ambiguous to me. `disallow` was clear to me, this package's purpose is to disallow access with some exceptions. Could we keep the name the same, or else change it to something a little more explicit like `apiaccess`?

Reply:
As we added more things to the package, I thought `disallow` no longer was a great name. For example, `disallow.AllowAll` looks odd. I agree that `expose` isn't much better. I like `access` or `apiaccess`, will adjust!